### PR TITLE
fix(react-core): updated class names to include versions

### LINKED ISSUE #9144: Clean up example/demo class names still missing versioned prefix
#### Components
```
packages/react-core//src/components/MenuToggle/examples/MenuToggle.css:#ws-react-c-menu-toggle-primary .pf-c-menu-toggle, #ws-react-c-menu-toggle-secondary .pf-c-menu-toggle {
packages/react-core//src/components/Alert/examples/alert.css:.ws-react-c-alert .pf-c-alert {
packages/react-core//src/components/Checkbox/examples/checkbox.css:.ws-react-c-checkbox .pf-c-check.nested {
packages/react-core//src/demos/JumpLinks.md:    const masthead = document.getElementsByClassName('pf-c-masthead')[0];
packages/react-core//src/demos/examples/JumpLinks/JumpLinksWithDrawer.js:    const masthead = document.getElementsByClassName('pf-c-masthead')[0];
```

#### Layouts
```
packages/react-core//src/layouts/Split/examples/split.css:.ws-react-l-split .pf-l-split,
packages/react-core//src/layouts/Split/examples/split.css:.ws-react-l-split .pf-l-split > .pf-l-split__item {
packages/react-core//src/layouts/Gallery/examples/gallery.css:.ws-react-l-gallery .pf-l-gallery {
packages/react-core//src/layouts/Gallery/examples/gallery.css:.ws-react-l-gallery .pf-l-gallery > div {
packages/react-core//src/layouts/Bullseye/examples/bullseye.css:.ws-react-l-bullseye .pf-l-bullseye,
packages/react-core//src/layouts/Bullseye/examples/bullseye.css:.ws-react-l-bullseye .pf-l-bullseye > div {
packages/react-core//src/layouts/Level/examples/level.css:.ws-react-l-level .pf-l-level {
packages/react-core//src/layouts/Level/examples/level.css:.ws-react-l-level .pf-l-level,
packages/react-core//src/layouts/Level/examples/level.css:.ws-react-l-level .pf-l-level > div {
packages/react-core//src/layouts/Stack/examples/stack.css:.ws-react-l-stack > .pf-l-stack {
packages/react-core//src/layouts/Stack/examples/stack.css:.ws-react-l-stack .pf-l-stack,
packages/react-core//src/layouts/Stack/examples/stack.css:.ws-react-l-stack .pf-l-stack > .pf-l-stack__item {
packages/react-core//src/layouts/Flex/__tests__/Flex.test.tsx:            .className.replace('pf-l-flex', '')
packages/react-core//src/layouts/Flex/examples/flex.css:.ws-react-l-flex .pf-l-flex {
packages/react-core//src/layouts/Flex/examples/flex.css:.ws-react-l-flex .pf-l-flex > div,
packages/react-core//src/layouts/Grid/examples/grid.css:.ws-react-l-grid > .pf-l-grid,
packages/react-core//src/layouts/Grid/examples/grid.css:.ws-react-l-grid > .pf-l-grid > .pf-l-grid__item {
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">100C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">50C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0C</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">3.6Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">1.5Ghz</div>
packages/react-core//src/demos/CardDemos.md:                          <div className="pf-l-flex__item">0GHZ</div>
```

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
**What**: Closes #9144

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/AboutModal/examples/AboutModal.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: About modal
 section: components
-cssPrefix: pf-c-about-modal-box
+cssPrefix: pf-v5-c-about-modal-box
```
Comment (thatblindgeye) [inline packages/react-core/src/components/AboutModal/examples/AboutModal.md:4]: Do we want to update this in example files right now? Looks like the CSS variables tables in component pages is empty in the preview. cc @nicolethoen would this be tied to https://github.com/patternfly/patternfly-org/issues/3489 ?

Comment (wise-king-sullyman) [reply packages/react-core/src/components/AboutModal/examples/AboutModal.md:4]: Yeah I think that with https://github.com/patternfly/patternfly-org/issues/3489 the tables will work with this change.

Comment (nicolethoen) [reply packages/react-core/src/components/AboutModal/examples/AboutModal.md:4]: agreed! I think to keep parity with core, we should use the versioned prefixes here.

**Review by nicolethoen — APPROVED** [review]

**Review by thatblindgeye — APPROVED** [review]

**Review by gitdallas — APPROVED** [review]

**Review by mcoker — APPROVED** [review]
LGTM, one small comment that isn't a blocker.

File: packages/react-core/src/demos/CardDemos.md [hunk 2]
```diff
@@ -1571,9 +1571,9 @@ CardNestedDemo = () => {
                           spaceItems={{ default: 'spaceItemsNone' }}
                           alignItems={{ md: 'alignItemsFlexEnd' }}
                         >
-                          <div className="pf-l-flex__item">100C</div>
-                          <div className="pf-l-flex__item">50C</div>
-                          <div className="pf-l-flex__item">0C</div>
+                          <div className="pf-v5-l-flex__item">100C</div>
+                          <div className="pf-v5-l-flex__item">50C</div>
+                          <div className="pf-v5-l-flex__item">0C</div>
```
[the remark this answers is not in the document]
Comment (wise-king-sullyman) [reply packages/react-core/src/demos/CardDemos.md]: Pushed a change that I think addressed this, look good to you?

**Pull request merged**